HW_3/Task_1.py

```diff
@@ -0,0 +1,12 @@
+# Дан список повторяющихся элементов. Вернуть список с дублирующимися элементами.
+# В результирующем списке не должно быть дубликатов.
+
+def double_list(array: list[int]) -> list[int]:
+    res = set()
+    for el in array:
```

Comment:
Для избежание повторения подсчетов. тут лучше использовать множество 
```suggestion
    for el in set(array):
```